chain/consensus/hierarchical/types.go

```diff
@@ -18,6 +18,11 @@ type ConsensusParams struct {
 	MinValidators uint64          // Min number of validators required to start a network.
 }
 
+type MiningParams struct {
```

Comment:
Should we call this something like `MiningLogParams`, or do you expect to use these params for something else in the future?

Reply:
I thought we will add other parameters in the future. So it's not only for log parameters

Reply:
Gotcha, the this is fine by me.